src/com/core/tictactoe/CommandLineUI.java

```diff
@@ -0,0 +1,62 @@
+package com.core.tictactoe;
+
+import java.io.InputStream;
+import java.io.PrintStream;
+import java.util.Scanner;
+
+public class CommandLineUI {
+
+    PrintStream output;
+    Scanner input;
+
+    CommandLineUI(){}
+
+    CommandLineUI(PrintStream output, InputStream input){
+        this.output = output;
+        this.input = new Scanner(input);
+    }
+
+    protected void showBoard(Board board) {
```

Comment:
If any of these 'protected' methods are to be used by a non-CommandLineUI class then I'd prefer them to be public.

Reply:
I'm not sure about the future stories, so can't answer your question. I'd prefer to change them to public only when it's necessary - is it ok to keep them as protected at this point?